changelogs/fragments/manageiq-resource_id.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - Added new parameter 'resource_id' to modules manage_tags and manageiq_policies. Parameter 'resource_id' can be used instead of parameter 'resource_name'.
```

```suggestion
  - manageiq_tags and manageiq_policies - added new parameter ``resource_id``. This parameter can be used instead of parameter ``resource_name`` (https://github.com/ansible-collections/community.general/pull/719).
```
(See https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)